openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/EntityLineage.component.tsx

```diff
@@ -479,6 +479,89 @@ const EntityLineageComponent: FunctionComponent<EntityLineageProp> = ({
     [nodes, updatedLineageData]
   );
 
+  /**
+   * take node and get the columns for that node
+   * @param expandNode
+   */
+  const getTableColumns = async (expandNode?: EntityReference) => {
+    if (expandNode) {
+      try {
+        const res = await getTableDetails(expandNode.id, ['columns']);
+        const tableId = expandNode.id;
+        const { columns } = res;
+        tableColumnsRef.current[tableId] = columns;
+        updateColumnsToNode(columns, tableId);
+      } catch (error) {
+        showErrorToast(
+          error as AxiosError,
+          `Error while fetching ${getDataLabel(
```

Comment:
Can we add localization here?

Reply:
We can add in a separate PR for now I just move the old function from bottom to top!

Reply:
done